src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -560,7 +562,7 @@ def handler(args):
 
         return CLIError('Wait operation timed-out after {} seconds'.format(timeout))
 
-    cmd = CliCommand(name, handler, arguments_loader=arguments_loader)
+    cmd = AzCliCommand(context.ctx, name, handler, arguments_loader=arguments_loader)
```

Comment:
Exactly.  It is exactly odd. Hence: https://github.com/Microsoft/knack/issues/13 😅